src/builtin/welcome.ts

```diff
@@ -9,7 +9,7 @@ const msg_types = Sphinx.MSG_TYPE
 
```

Comment:
same as other `src/builtin` files

Reply:
what i changed in these files are javascript mechanics, such as adding `{` and `}` around a case statement because eslint complains about a constant being accessible from outside it, or i moved it entirely in 1 return statement to not declare a const